.github/workflows/dev-to-sync.yml

```
@@ -48,3 +48,14 @@ jobs:
           git config --global user.name "GitHub Action"
           git add -A
           git diff --quiet && git diff --staged --quiet || (git commit -m 'Update blog content'; git push)
+
+  build:
+    uses: ./.github/workflows/build.yml
+    # with:
+    #   HUGO_VERSION: '0.134.2'
+
+  deploy:
+    uses: ./.github/workflows/deploy.yml
```

actionlint

could not read reusable workflow file for "./.github/workflows/deploy.yml": open /cache/34/3e8366bc-f1cc-4f09-b4ec-fa88c0ec5b90/home/jailuser/git/.github/workflows/deploy.yml: no such file or directory

(workflow-call)